PullRequestCreated/PullRequestCreated.ts

```diff
@@ -5,16 +5,24 @@ import { JIRA_ISSUE_PATTERN, JIRA_TASK_ISSUE } from '../lib/Constants';
 class PullRequestCreated extends OctokitAction {
   protected async execute(): Promise<void> {
     const pr = await this.getPullRequest(this.payload.pull_request.number);
-    if (this.shouldCreateIssue(pr)) {
+    const linkedIssues = pr?.title?.match(JIRA_ISSUE_PATTERN) || null;
+    if (pr != null && linkedIssues == null) {
       const projectKey = this.getInput('jira-project');
       const issueKey = await this.jira.createIssue(projectKey, JIRA_TASK_ISSUE, pr.title);
       await this.updatePullRequestTitle(this.payload.pull_request.number, `${issueKey} ${pr.title}`);
       await this.updatePullRequestDescription(this.payload.pull_request.number, `${issueKey}\n\n${pr.body || ''}`);
+    } else if (pr != null) {
+      const mentionedIssues = this.findMentionedIssues(pr);
+      const notMentionedIssues = linkedIssues.filter(issue => !mentionedIssues.includes(issue));
+      console.log(`Adding the following ticket in description: ${notMentionedIssues}`);
+      if (notMentionedIssues.length > 0) {
+        await this.updatePullRequestDescription(this.payload.pull_request.number, `${notMentionedIssues.join('\n')}\n\n${pr.body || ''}`);
```

Comment:
Relying on the auto-linking GH feature was not a good idea because it adds additional complexity to the overall solution. And if somebody links a project form a non-configured autolink, like BUILD-1234, it would not work.

Please add links manually with domain, and only the issueKey as the visible part